app/src/main/java/com/github/multimatum_team/multimatum/AddDeadlineActivity.kt

```diff
@@ -78,20 +92,38 @@ class AddDeadlineActivity : AppCompatActivity() {
             TimePickerDialog.OnTimeSetListener { _, hoursOfDay, minutes ->
                 selectedDate = LocalDateTime.of(
                     selectedDate.year, selectedDate.monthValue, selectedDate.dayOfMonth,
-                    hoursOfDay, minutes)
+                    hoursOfDay, minutes
+                )
 
                 textTime.text = selectedDate.toLocalTime().toString()
             }
 
         // Prepare the datePickerDialog
-        val timePickerDialog = TimePickerDialog(this, timeSetListener,
-            selectedDate.hour, selectedDate.minute, true)
+        val timePickerDialog = TimePickerDialog(
+            this, timeSetListener,
+            selectedDate.hour, selectedDate.minute, true
+        )
 
         // Show the Dialog on the screen
         timePickerDialog.show()
 
     }
 
+    val checkBoxIdTime = mapOf<Int, Long>(
+        R.id.radio_notification_1h to Duration.ofHours(1).toMillis(),
+        R.id.radio_notification_5h to Duration.ofHours(5).toMillis(),
+        R.id.radio_notification_1d to Duration.ofDays(1).toMillis(),
+        R.id.radio_notification_3d to Duration.ofDays(3).toMillis()
+    )
```

Comment:
That is a suggestion and I am not sure whether it is feasible, but as you have many different data relative to these reminders, you should think of creating a data class, something like
```Kotlin
data class Reminder(delayMillis: Long, delayText: String, resourceId: Int)
...
val reminder1Hour = Reminder(Duration.ofHours(1).toMillis(), "1 hour", R.id.radio_notification_1h)
```
and put all the reminders into a single list. This would be less error-prone and make it easier to add reminders. And it would suppress the need for the constants that I said in my previous comment.